### PR TITLE
Ap/clean documents

### PR DESCRIPTION
**Pull request by alimosaed**
**Changes Made**

- Updated documents.

**Testing**

- No code has changed.

**Documentation**

- Updated readme files.

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This guide will help you get started with the Solace AI Event Connector.
 
 ## Prerequisites
 
-- Python 3.10 or later
+- Python 3.10.x, 3.11.x or 3.12.x
```
Comment (cyrus2281) [inline docs/getting_started.md:7]: Isn't this just the same as later? are we not going to support `3.13`?

**Review by cyrus2281 — COMMENTED** [review]
Added a comment

File: docs/getting_started.md (hunk 1 above)
Comment (alimosaed) [reply docs/getting_started.md:7]: Some Python packages in the project are not compatible with versions 3.13 and upper.

Comment (alimosaed) [reply docs/getting_started.md:7]: The connector was not compatible with Version 3.13. However, I tested again and it seems that 3.13 supports the connector as well. Reverted this change.

**Review by cyrus2281 — APPROVED** [review]
Looks good

**Pull request merged**